src/devices/machine/z80scc.h

```diff
@@ -631,6 +631,8 @@ class z80scc_channel : public device_t,
 	// SCC specifics
 	int m_ph;       // Point high command to access regs 08-0f
 	uint8_t m_zc;
+
+	void check_waitrequest();
```

Comment:
Can you please move this up to be with the other private member function declarations?  Intermixing member functions and data members can get confusing.